src/intent.html

```diff
@@ -526,6 +526,7 @@ <h3 id="mixing_intent_examples">Intent Examples</h3>
   &lt;mo arg="script">&#x2032;&lt;/mo>
 &lt;/msup>
 </pre>
+<blockquote>x prime<br/>x superscript prime end superscript</blockquote>
```

Comment:
I think this should just have `x prime`  (which I just checked also matched what mathcat does) I don't see how you can get `x superscript prime` as the intent masks the presentation so you are explicitly stoping the "superscript" being read (if you are using the intent at all) The system may of course have an option to read the underlying markup ignoring intent but that shouldn't be in this "intent examples" section.

Reply:
Right, exactly, when the intent is used it is `x prime`. That's the starting fragment before the `<br/>`.

The second entry is the speech when intent isn't used - I copied it directly from the example right above, there are several that follow that pattern.